stats/Record.md

```diff
@@ -68,5 +72,14 @@ MeasurementMap measurementMap = new MeasurementMap();
 measurementMap.put(RPC_LATENCY, 10.3);
 measurementMap.put(RPC_BYTES_SENT, 124);
 measurementMap.record();  // reads context from thread-local.
+
+// Another example on recording against sampled SpanContext.
+SpanContext spanContext = tracer.getCurrentSpan().getContext();
+if (spanContext.getTraceOptions().isSampled()) {
+  Map<String, String> map = new HashMap<>();
+  // Client code needs to take care of encoding.
+  map.put("TraceId", encode(spanContext.getTraceId()));
+  map.put("SpanId", encode(spanContext.getSpanId()));
+  measurementMap.record(tagContext, map);
```

Comment:
can someone mention how an existing metrics system might handle this data as it is inherently unlimited cardinality. For example, is there an exemplar feature somewhere? I get where this is going, but bottom-up I'm kindof guessing.

Reply:
@adriancole here are some examples of data we want to record:
1) TraceId/SpanId this will allow the backend to display on their UI information about traces within a percentile (usually) (e.g. "Here are 99%tile traces").
2) Dropped tags. Let's assume that in the tag map/set/context we have 20 unique tags and we break down stats only by 4 of them. We record the rest of 16 as dropped tags. This can be used as example of requests tags for different percentiles.

The first example is what we call correlation between metrics and traces.